backend/src/main/java/apsi/team3/backend/DTOs/LoggedUserDTO.java

```diff
@@ -3,14 +3,16 @@
 import apsi.team3.backend.model.UserType;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.ToString;
 
 @Getter
 @ToString
 @EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
 public class LoggedUserDTO extends UserDTO {
-    private final String authHeader;
-    private final String type;
+    private String authHeader;
```

Comment:
czemu to zmieniamy?

Reply:
tak jak pisałem na discordzie, jeśli są pola final to nie da się stworzyć konstruktora bezargumentowego,  a ten z kolei jest niezbędny do działania deserializacji, która jest wykorzystywana w testach

Reply:
oki, umknęła mi ta dyskusja